homeassistant/components/mikrotik/config_flow.py

```diff
@@ -100,6 +102,12 @@ async def async_step_device_tracker(self, user_input=None):
                 CONF_ARP_PING,
                 default=self.config_entry.options.get(CONF_ARP_PING, False),
             ): bool,
+            vol.Optional(
+                CONF_SCAN_INTERVAL,
+                default=self.config_entry.options.get(
+                    CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL
+                ),
+            ): int,
```

Comment:
Usually we recommend setting an update interval that the device can reasonably support, and if someone wants more frequent updates, they can use the `homeassistant.update_entity` service (example automation here: https://next.home-assistant.io/integrations/hunterdouglas_powerview/#force-update-shade-position) to trigger additional polling as long as `async_update/update` is implemented on the entity.

Reply:
So you are recommending to set a fixed update interval. What is the recommendation for device_tracker integrations?

Reply:
The built-in default update interval is 15s https://github.com/home-assistant/core/blob/16dd70ba994b5f9cb50a7d229e97ea8629740ef9/homeassistant/helpers/entity_component.py#L30

Ideally we want to balance getting the data updated quickly with not overloading the device.  The minimum allowed is 5s to prevent overloading the system.  Assuming the device handles the new value 10s without getting overloaded that seems fine to me.  

The polling should never be quicker that the device handle or too frequent that a loaded Home Assistant instance is always tying up an executor thread. If we have too many integrations with a high polling frequency all using the executor, the whole system can grind to a halt since there is a [limit of 64 executor jobs](https://github.com/home-assistant/core/blob/16dd70ba994b5f9cb50a7d229e97ea8629740ef9/homeassistant/runner.py#L27) running at once since threads performance does not scale linearly.

Reply:
Thanks for the information. I will set the default to 15 and set the minimum to 10.

Reply:
For clarity, I'm suggesting removing the option and letting users create an automation if they want a faster update interval.

The update interval option is a frequent code review discussion.  This post does a much better job of explaining it than I just did:

https://github.com/home-assistant/core/issues/71563#issuecomment-1126996836

Reply:
Now its clear. I tested the `update_entity` service and it is working fine with mikrotik entities. So I will revert the update interval to 10s and remove the config option.